modules/govuk/manifests/node/s_asset_base.pp

```diff
@@ -99,4 +128,71 @@
     hour    => 5,
     minute  => 25,
   }
+
+  file { '/usr/local/bin/process-uploaded-attachments.sh':
+    content => template('govuk/node/s_asset_base/process-uploaded-attachments.sh.erb'),
+    mode    => '0755',
+  }
+
+  file { '/usr/local/bin/copy-attachments.sh':
+    content => template('govuk/node/s_asset_base/copy-attachments.sh.erb'),
+    mode    => '0755',
+  }
+
+  if $s3_bucket {
+
+    package { 's3cmd':
+      ensure   => 'present',
+      provider => 'pip',
+    }
+
+    file { '/home/assets/.s3cfg':
+      ensure  => present,
+      content => template('govuk/node/s_asset_base/s3cfg.erb'),
+      owner   => 'assets',
+      group   => 'assets',
+    }
+
+    file {
+    [ '/etc/govuk/aws', '/etc/govuk/aws/env.d']:
+      ensure  => directory,
+      owner   => 'assets',
+      group   => 'assets',
+      mode    => '0750';
+    '/etc/govuk/aws/env.d/AWS_ACCESS_KEY_ID':
+      ensure  => present,
+      content => $aws_access_key_id,
+      owner   => 'assets',
+      group   => 'assets',
+      mode    => '0640';
+    '/etc/govuk/aws/env.d/AWS_SECRET_ACCESS_KEY':
```

Comment:
Do we do this explicitly with `file` and not `govuk_envvar` so we can limit the permissions?